hibernate-core/src/main/java/org/hibernate/persister/entity/EntityPersister.java

```diff
@@ -798,7 +814,11 @@ Object createProxy(Serializable id, SharedSessionContractImplementor session)
 	EntityTuplizer getEntityTuplizer();
 
 	BytecodeEnhancementMetadata getInstrumentationMetadata();
-	
+
+	default BytecodeEnhancementMetadata getBytecodeEnhancementMetadata() {
+		return getInstrumentationMetadata();
+	}
```

Comment:
```getBytecodeEnhancementMetadata()``` does not seem to be used anywhere. Should it be removed?

Reply:
No.  Actually the other way around - that is the method we should use.  I got caught not finishing what I intended there.  I have changed that (and some other things) and will push soon